modules/ROOT/pages/fd-connection-view-audit-logs.adoc

```diff
@@ -0,0 +1,24 @@
+= View Audit Logs for Connections That are Shared
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.
+
+To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:
+
+* The time and date of the action
+* The product in which the action was taken (i.e., Flow Designer)
+* The type of change, which is always "Connection"
+* The type of action: Create, Edit, or Delete,
+* The type of connector that the connection was for, such as the Database connector.
```

Comment:
I'm not sure about the "Type of connector" phrase, I mean, you can see it, but only downloading the payload. 
In the main table, in the column "Object" you will see the "name" of the connection, 

that name is written by the user so, no necessarily you will see a name related to the type of connector...

Reply:
I misread the text that appears in this column in the GIF that you sent me. The text is "Amazon S3 connection," but I read it as "Amazon S3 connector."
Changed.